tests/sentry/web/frontend/test_auth_organization_login.py

```diff
@@ -997,3 +936,111 @@ def test_correct_redirect_as_2fa_user_invited_member(self):
         )
 
         assert resp.redirect_chain == [("/auth/2fa/", 302)]
+
+
+class OrganizationAuthLoginNoPasswordTest(AuthProviderTestCase):
+    def setUp(self):
+        self.owner = self.create_user()
+        self.organization = self.create_organization(name="foo", owner=self.owner)
+        self.user = self.create_user("bar@example.com", is_managed=False, password="")
+        self.auth_provider = AuthProvider.objects.create(
+            organization=self.organization, provider="dummy"
+        )
+        self.path = reverse("sentry-auth-organization", args=[self.organization.slug])
+        self.auth_sso_path = reverse("sentry-auth-sso")
+        UserEmail.objects.filter(user=self.user, email="bar@example.com").update(is_verified=False)
+
+    def test_flow_authenticated_without_verified_without_password(self):
+        """
+        Given an existing authenticated user, and an updated identity (e.g.
```

Comment:
I don't believe this test is doing what this comment suggests.

Reply:
The comment originates with 1361e383e5ab4b5c1a2782d22efc53ded57602bc. It isn't covering the behavior of the feature we're currently working on (it was only moved).

I believe it is, however, covering the exact same preconditions, only without the `organizations:idp-automatic-migration` flag being set. So, the old behavior that we're about to replace.

Reply:
> Given an existing authenticated user, and an updated identity (e.g. the ident changed from the SSO provider), we should be re-linking the identity automatically as they don't have a password.

I see, it seems that the original comment then is off on that particular case, as it appears the case was copy pasted (same comment is on 3 other cases). we should just delete it as it makes no sense.
1.  the user isn't authenticated in this test
2. there is no "previous identity" in this test
3. we don't test that any identity is "re-linked" here.